core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/SnowOwlApiConfig.java

```diff
@@ -214,18 +205,20 @@ protected void addFormatters(FormatterRegistry registry) {
 	}
 	
 	@Bean
-	public MethodValidationPostProcessor methodValidationPostProcessor() {
+	public static MethodValidationPostProcessor methodValidationPostProcessor() {
 		return new MethodValidationPostProcessor();
 	}
 	
 	@Bean
 	public MultipartResolver multipartResolver() {
-		final HttpConfig httpConfig = ApplicationContext.getInstance().getService(SnowOwlConfiguration.class).getModuleConfig(ApiConfiguration.class).getHttp();
-	    final CommonsMultipartResolver multipartResolver = new CommonsMultipartResolver();
-	    multipartResolver.setMaxUploadSizePerFile(httpConfig.getMaxFileSizeBytes());
-	    multipartResolver.setMaxUploadSize(httpConfig.getMaxRequestSizeBytes());
-	    multipartResolver.setMaxInMemorySize(httpConfig.getMaxInMemorySizeBytes());
-	    return multipartResolver;
+		// TODO add back max file size restrictions somehow
```

Comment:
Looks like this is a Servlet-level configuration parameter now:

> Note: In order to use Servlet container based multipart parsing, you need to mark the affected servlet with a "multipart-config" section in web.xml, or with a `MultipartConfigElement` in programmatic servlet registration, or (in case of a custom servlet class) possibly with a `MultipartConfig` annotation on your servlet class. Configuration settings such as maximum sizes or storage locations need to be applied at that servlet registration level; a Servlet container does not allow for them to be set at the `MultipartResolver` level. 

* [Upgrading to Spring Framework 6.x](https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-6.x#web-applications-1)
* Javadoc of [StandardServletMultipartResolver](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/multipart/support/StandardServletMultipartResolver.html) (quote source)

Reply:
I think this also means that the method can be removed entirely.

Reply:
Yes, thanks. I haven't got enough time to dig into this yet, but will do so now before merging the PR.

Reply:
Done